src/tests/e2etest/Tests.cs

```diff
@@ -0,0 +1,126 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using NUnit.Framework;
+using Microsoft.Azure.Devices.Shared;
+using System.Text.Json;
+using System;
+using System.Threading.Tasks;
+
+namespace e2etesting
+{
+    public class CommandRunnerTests : E2eTest
+    {
+        public enum Action
+        {
+            ActionNone = 0,
+            ActionReboot,
+            ActionShutdown,
+            ActionRunCommand,
+            ActionRefreshCommandStatus
```

Comment:
Please update for current CommandRunner.

Reply:
CommandRunner v2:

[https://github.com/Azure/iot-plugandplay-models/blob/main/dtmi/osconfig/commandrunner-2.json](https://github.com/Azure/iot-plugandplay-models/blob/main/dtmi/osconfig/commandrunner-2.json)

Note there are new Action values there.

Same for the State below.